Makefile-test.mk

```diff
@@ -151,6 +154,15 @@ run-test-tas-e2e-%: FORCE
 		./hack/e2e-test.sh
 	$(PROJECT_DIR)/bin/ginkgo-top -i $(ARTIFACTS)/$@/e2e.json > $(ARTIFACTS)/$@/e2e-top.yaml
 
+run-test-queuename-e2e-%: K8S_VERSION = $(@:run-test-queuename-e2e-%=%)
```

Comment:
```suggestion
run-test-withoutqueuename-e2e-%: K8S_VERSION = $(@:run-test-queuename-e2e-%=%)
```
Maybe like this?

Reply:
Same as before, I think this name depends on the final strategy we follow

Reply:
Yes, `withoutqueuename` would not be best - because it would depend on the actual test.
I think a generic name as "customconfigs" is ok.